docs/update-syft.md

```diff
@@ -0,0 +1,51 @@
+﻿# Updating Syft
+
+For container detection we use [Syft][1].
```

Comment:
Probably worth leaving a note here that this can only be done by maintainers, not by all users.

Reply:
IMO it should be obvious that not everyone can push container images to our container registry.